test/test_env.py

```diff
@@ -1946,6 +1946,32 @@ def test_rollout_policy(self, batch_size, rollout_steps, count):
             else:
                 assert (td["lazy"][..., i]["action"] == 0).all()
 
+    @pytest.mark.parametrize("batch_size", [(), (1, 2), (2,)])
+    @pytest.mark.parametrize("env_type", ["serial", "parallel"])
+    def test_vec_env(self, batch_size, env_type, rollout_steps=4, n_workers=2):
+        env_fun = lambda: HeteroCountingEnv(batch_size=batch_size)
+        if env_type == "serial":
+            vec_env = SerialEnv(n_workers, env_fun)
+        else:
+            vec_env = ParallelEnv(n_workers, env_fun)
+        vec_batch_size = (n_workers,) + batch_size
+        # check_env_specs(vec_env, return_contiguous=False)
+        policy = HeteroCountingEnvPolicy(vec_env.input_spec["_action_spec"])
```

Comment:
Note to self: `input_spec["_action_spec"]` is clunky, we should have some other way to access the composite spec of the action.

Reply:
+1